documentation/Installation.md

```diff
@@ -20,7 +20,7 @@ Saffire. Once the box has started, issue the following:
 
     vagrant ssh
     cd /vagrant
-    sh build.sh
+    sudo ./build.sh -i
```

Comment:
I noticed that sh and bash are not symlinked in the virtualbox, and causes issues (`[[` for instance, is a bash extension).  It should be doable to just run  `./build.sh`.

The `-i` option is for explicitly installing saffire in `/usr/local/bin/saffire`, for which sudo is needed, but considering we are referencing `/vagrant/build/release/bin/saffire`, we do not need it.

@Drarok:  could you change this to:  `./build.sh`  instead. I'd reckon it would work (it does here, anyway), OR, reference to `/usr/local/bin/saffire` below (both are fine by me).